### PR TITLE
Core kit 2.3.5

### PR DESCRIPTION
**Pull request by larvacea**
**Description of changes:**

* Update Twoliter.toml and Twoliter.lock for 2.3.5
* Update CHANGELOG.md for 2.3.5

**Testing done:**

Core kit builds, bob can build a variant.

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: CHANGELOG.md
Comment (yeazelm) [inline CHANGELOG.md]: We can either list the versions explicitly or leave the implication it applies to all. 
```suggestion
 * Fix issue where a null value would fail to render the credential provider template for Kubernetes ([#101])
```

File: CHANGELOG.md
Comment (yeazelm) [inline CHANGELOG.md]: The first and third were in the same PR, let's talk about them together.
```suggestion
 * Improve EBS volume udev rules by adding a symlink to `/dev/by-ebs-id` and remove `/dev/` the device name returned by ghostdog ([#98])
 * Update kernels to 5.10.223-212 and 6.1.102-111 ([#99])
```
Or we can keep them as two lines, but the PR is the same, it makes this seem like they were done separately as is.

File: CHANGELOG.md
Comment (yeazelm) [inline CHANGELOG.md]:
```suggestion
 * Add collect-kernel-config script to tools ([#84])
```

**Review by KCSesh — APPROVED** [review]
LGTM - after 101 merges.

**Review by KCSesh — APPROVED** [review]

**Review by yeazelm — APPROVED** [review]

**Pull request merged**
